qemu/tests/cfg/cpu_device_hotplug_during_boot.cfg

```diff
@@ -0,0 +1,13 @@
+- cpu_device_hotplug_during_boot: cpu_device_hotpluggable
+    required_qemu = [2.6.0, )
+    virt_test_type = qemu
+    type = cpu_device_hotplug_during_boot
+    qemu_sandbox = on
+    vcpu_devices = vcpu1
+    only Linux
```

Comment:
Why only linux?

Reply:
1. I don't think we have a solution to verify if the windows guest is in boot stage
2. Does windows guest support hotplug vCPU in boot stage?

Reply:
It should be a rare scenario in real product env to hotplug cpu in boot stage. After a second thought, I'm ok to skip windows guest, thanks.

Reply:
It should be a rare scenario in real product env to hotplug cpu in boot stage. After a second thought, I'm ok to skip windows guest, thanks.